### ISSUE TITLE
Shortcuts for smooth workflow

### ISSUE THREAD

**Issue by Stefanoko** [olivierkes/manuskript#626]
As a writer I prefer the keyboard over mouse or touchpad. I am on Version 0.9.0 for Windows. A typical scenario:

Create a new item: I am forced to use the mouse. (Btw, the new item does not get the focus, although I just created it, I cannot name by typing right away)

To open my selected item, I try the usual TAB or ENTER but nothing seems to work other than using the mouse again.
Same for going back up to its container, there ís a button at the bottom left to do that, but no key shortcut. Mouse again.

Ctrl+Tab to switch tabs brings up the snowflake guideline instead.
Ctrl+W to close current tab does not work, either

Ctrl+S does not seem to save my work. 

Also missing are shortcuts to

- show/hide/access the project tree
- open item in fullscreen
- toggle focus mode on/off (to use it temporarily and not as a basic setting)

Please add more shortcuts, at least for the bread+butter tasks. This is a writer's software, after all. Hacking away at the keyboard is what we do

**Comment by arcandio** [comment]
Agreed, I find the ability to create and edit an outline using the keyboard to be a dealbreaker for me.
